examples/policy/sandbox-automount-token-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
```

Comment:
Also do we expect more policies in this directory ? 
If not would a directory something like make sense: examples/token-binding-policy ?

Reply:
Yes, there will be at least one more, for example, #33